cartridges/openshift-origin-cartridge-haproxy/bin/control

```diff
@@ -29,7 +29,7 @@ function isrunning() {
 function ping_server_gears() {
     #  Ping the server gears and wake 'em up on startup.
     for geardns in $(web_gears | cut -f 3 -d ','); do
-          [ -z "$geardns" ]  ||  curl "http://$geardns/" > /dev/null 2>&1  ||  :
```

Comment:
Eek, I think you want something along the lines of `(curl "http://$geardns/" > /dev/null 2>&1 &)` (which will run curl in the background in a subshell) rather than using backticks (which will try to execute the output of the curl command).